project/dexter-monitor/routes/server.js

```diff
@@ -34,17 +34,39 @@ const _ = require('lodash');
 let serverList = [];
 let serverListLastModifiedTime = new Date();
 let checkingServerStatus;
+let doNotStartChecking;
 
-exports.init = function(doNotStartChecking){
+exports.init = function(_doNotStartChecking){
 
     mailing.init();
 
-    loadServerList()
+    doNotStartChecking = _doNotStartChecking;
+    updateServerListInternal();
```

Comment:
`doNotStartChecking` flag is just for unit testing, so it always has `undefined` value for production use. In unit testing, the flag is set to `true` not to call server heartbeat checking code.